config/locales/articles.en.yml

```
@@ -0,0 +1,18 @@
+en:
+  articles:
+    paths:
+      new: 'New article'
+      edit: 'Edit'
+    actions:
+      edit: 'Edit'
+      remove: 'Remove'
+      create: 'Create'
+    edit:
+      title: 'Edit article'
+      success: 'Article edited'
+    new:
+      title: 'Enter new article'
+      success: 'Article created'
+    list:
+      footer_prefix: "Posted"
+      footer_postfix: "ogo"
```

You can do something like this:

```suggestion
      posted_ago:
        html: "<span>Posted</span> %{time} <span>ago</span>"
```

and use it like this
```ruby
t(
  "articles.list.posted_ago.html",
  time: tag.time(
    time_ago_in_words(article.created_at),
    datetime: artcile.created_at.iso8601
  )
)
```

You can move this thing to a helper if it seems complicated :)

---

I didn't know that I can use HTML in translations